test/canvas.test.js

```diff
@@ -1492,4 +1492,53 @@ describe('Canvas', function () {
     assert.ok(ctx.isPointInPath(3, 3, 'evenodd'));
   });
 
+  it('Context2d#rotate(angle)', function () {
+    var canvas = createCanvas(4, 4);
+    var ctx = canvas.getContext('2d');
+
+    // Number
+    ctx.resetTransform();
+    testAngle(1.23);
+
+    // String
+    ctx.resetTransform();
+    testAngle('-4.56e-1', -0.456);
+
+    // Boolean
+    ctx.resetTransform();
+    testAngle(true, 1);
+
+    // Array
+    ctx.resetTransform();
+    testAngle([7.8], 7.8);
+
+    // Object
+    var obj = {[Symbol.toPrimitive](){ return 0.89; }};
+    ctx.resetTransform();
+    testAngle(obj, 0.89);
+
+    // NaN
+    ctx.resetTransform();
+    ctx.rotate(0.91);
+    testAngle(NaN, 0.91);
+
+    // Infinite value
+    ctx.resetTransform();
+    ctx.rotate(0.94);
+    testAngle(-Infinity, 0.94);
+
+    function testAngle(angle, expected = angle){
```

Comment:
Default arguments are not supported on Node.js 4 which we currently are supporting

Reply:
@LinusU . Already fixed.